src/main.h

```diff
@@ -113,7 +113,7 @@ static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 1000000;
 /** Additional block download timeout per parallel downloading peer (i.e. 5 min) */
 static const int64_t BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 500000;
 /** Timeout in secs for the initial sync. If we don't receive the first batch of headers */
-static const uint32_t INITIAL_HEADERS_TIMEOUT = 30;
+static const uint32_t INITIAL_HEADERS_TIMEOUT = 120;
```

Comment:
buip055 test was failing because of timeout on my test setup, when I used a rotating disk.  But working fine using a ramdisk.  I talked to @ptschip and we agreed that its ok to increase this value.